src/twisted/internet/defer.py

```diff
@@ -167,17 +167,53 @@ def execute(
         return succeed(result)
 
 
+def _isAsyncDefValue(o: Any) -> bool:
```

Comment:
This is sort of moot in this specific case, but in general if you can get away with `o: object`  rather than `o: Any` you should generally go that way